docs/source/quickstart.md

```diff
@@ -135,4 +135,51 @@ python -u "$@" --node_rank $SLURM_NODEID -u ${PATH_TO_MAMMOTH}/train.py \
     -tensorboard -tensorboard_log_dir ${LOG_DIR}/${EXP_ID}
 ```
 
-A complete example of training on the Europarl dataset is available at [MAMMOTH101](examples/train_mammoth_101.md).
\ No newline at end of file
+Explanation of Command:
```

Comment:
given this is a quickstart, we should probably showcase a single-node or single-GPU example, and add a sentence stating that
> MAMMOTH is specifically designed for distributed training of modular systems in multi-GPUs SLURM environments. 

Maybe we could have a separate distributed training page discussing that, along with the command explanation?